File: hw/lomas/u2/hw14DBMongoandGUI/hw14DBMongoandGUI.java/BillingSystem/src/main/java/ec/edu/espe/Billing/model/Payment.java
package ec.edu.espe.Billing.model;

public class Payment {
    private String id;
    private String paymentMethod;
    private double amount;

    public Payment(String id, String paymentMethod, double amount) {
        this.id = id;
        this.paymentMethod = paymentMethod;
        this.amount = amount;
    }

    public String getId() { return id; }
    public void setId(String id) { this.id = id; }
    public String getPaymentMethod() { return paymentMethod; }
    public void setPaymentMethod(String paymentMethod) { this.paymentMethod = paymentMethod; }
    public double getAmount() { return amount; }
    public void setAmount(double amount) { this.amount = amount; }
}

File: hw/lomas/u2/hw14DBMongoandGUI/hw14DBMongoandGUI.js/model/payment.js
class Payment {
    constructor(paymentId, paymentMethod, amount) {
        this.paymentId = paymentId;
        this.paymentMethod = paymentMethod;
        this.amount = amount;
    }

    toDict() {
        return {
            _id: this.paymentId,
            paymentMethod: this.paymentMethod,
            amount: this.amount
        };
    }
}

module.exports = Payment;